packages/core/src/routes/admin-user.ts

```diff
@@ -150,4 +151,20 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
       return next();
     }
   );
+
+  router.delete(
```

Comment:
not sure whether we should use `delete` or `patch` here?
this is a delete action of a customized type and is actually replacing the value of property `customData` with an empty placeholder.